pkg/scheduler/scheduler_test.go

```diff
@@ -613,6 +644,24 @@ func TestSchedule(t *testing.T) {
 				"eng-beta/needs-to-borrow",
 			},
 		},
+		"can borrow in cohort when lending limit enabled": {
```

Comment:
Better to refactor this testcase that can't borrow when enabled the lendingLimit.